contracts/StablecoinConverter.sol

```diff
@@ -312,6 +312,11 @@ contract StablecoinConverter is EpochTokenLocker {
         return elements;
     }
 
+    function acceptingSolutions(uint32 batchIndex) public view returns (bool) {
+        // Note that (now % 300 < 240) corresponds to 4 minutes after the batch has closed.
+        return batchIndex == getCurrentBatchId() - 1 && now % 300 < 240;
```

Comment:
Some people would probably say this should be a constant. But I'm not sure given that it's only used once it might as well stay here.

Reply:
nit: from what I understand `now % 300`  is the time of the current epoch where 300 is the epoch time (or `BATCH_TIME`), shouldn't this live in `EpockTokenLocker` then? Maybe something like:
```
// StableCoinConverter.sol
    function acceptingSolutions(uint32 batchIndex) public view returns (bool) {
         return getBatchTime(batchIndex) < 240;
    }

// EpockTokenLocker
    function getBatchTime(uint32 batchIndex) public view returns(uint256) {
        // ...
    }
```

That would also future proof it a bit where if `BATCH_TIME` ever changes then this won't have to be manually updated as well.

Reply:
I think since the `StablecoinConverter` is inheriting `EpockTokenLocker`, we can use directly `BATCH_TIME`

Reply:
I agree with @nlordell that by using a `getBatchTime()` method we are not leaking the implementation detail of how batches are calculated (based on `now` rather than some fixed start time for the first batch) to the StablecoinConverter.
This way we could change the batch timing implementation under the hood without affecting the StablecoinConverter, which seems cleaner to me.

Reply:
My only question here (regarding `getBatchTime`) is what are we expecting to be returned if the requested `batchIndex` has already passed. Perhaps `getBatchTime` should not accept any input parameter and simply return however many seconds have passed in the current batch. 

I guess this would be 

```
    function getBatchTime() public view returns(uint256) {
        return now % BATCH_TIME;
    }
```

Reply:
Applied these really great suggestions! Please check it out!

Reply:
Looking at the code now, we could just reuse the `getSecondsRemainingInBatch` method (I forgot this exsited) and just make sure it's more than 60s remaining? But up to you.